cdk/README.md

```diff
@@ -8,241 +8,155 @@ In this example we'll configure one Consul server in VPC with TLS and gossip enc
-```
-Once it's deployed, take note of the following from the output:
-* VPC id of the new VPC created by the CloudFormation stack
-* Security group id of the Consul server
-* ARN of the two AWS Secrets Manager secrets for Agent CA and Gossip. Use AWS console / CLI, this output is not visible on the CloudFormation stack.
-
-You can use the string `ConsulSshTunnel` from the CloudFormation output to create SSH tunnel to the Consul server and then access it's UI from http://localhost:8500/ui/
-
-## Step 2 - Create the sample CDK application
-
-### Create project directory
-Create an empty directory on your system, initialize Typescript CDK project and install NPM packages.
+### Step 1: Create the project directory
+First create an empty directory on your system, initialize Typescript CDK project and install NPM packages.
 
 ```
-mkdir app1
-cd app1
+mkdir app && cd app
 cdk init --language typescript
+cdk bootstrap aws://{ACCOUNT}/{REGION}
+
 npm install @aws-cdk/core @aws-cdk/aws-ec2 @aws-cdk/aws-ecs @aws-cdk/aws-secretsmanager @aws-cdk-containers/ecs-service-extensions @aws-quickstart/ecs-consul-mesh-extension
 npm update
 ```
 
-### Update app entry point 
-Edit your `bin/app1.ts` file by specifying the target AWS account id and the region. Update `$AWS_ACCOUNT_ID` and `$AWS_REGION` to match your target account and region.
-
-```
-#!/usr/bin/env node
-import 'source-map-support/register';
-import * as cdk from '@aws-cdk/core';
-import { App1Stack } from '../lib/app1-stack';
-
-const env = { account: '$AWS_ACCOUNT_ID', region: '$AWS_REGION' };
-
-const app = new cdk.App();
-new App1Stack(app, 'App1Stack', {env: env});
-```
-
-### Add extension to your stack
-Edit your stack `lib/app1-stack.ts`. First replace the `$MY_VPC_ID` and `$AWS_REGION` with the VPC id and the region from the CloudFormation stack that you deployed on step-1 earlier. Next you need to replace the `$CONSUL_SG` with the Consul server security group from step-1.
-
-
-```
-import * as cdk from '@aws-cdk/core';
-import * as ec2 from '@aws-cdk/aws-ec2';
-import * as ecs from '@aws-cdk/aws-ecs';
-import * as secretsmanager from '@aws-cdk/aws-secretsmanager';
-import { AssignPublicIpExtension, Container, Environment, Service, ServiceDescription, HttpLoadBalancerExtension } from '@aws-cdk-containers/ecs-service-extensions';
-import { ECSConsulMeshExtension, RetryJoin } from '@aws-quickstart/ecs-consul-mesh-extension';
+## Step 2: Create and Deploy the VPC Environment 
+Next we're going to create the VPC Environment for launching the Consul server and the microservices.
+You will need to use your public IP and it's CIDR set to `$ALLOWED_IP_CIDR`.
+Example: `ALLOWED_IP_CIDR=$(curl -s ifconfig.me)/32`
 
-export class App1Stack extends cdk.Stack {
+### Create a new file `lib/environment.ts`
+```ts
+export class Environment extends cdk.Stack {
```

Comment:
Missing imports
```
import * as ec2 from '@aws-cdk/aws-ec2';
import * as cdk from '@aws-cdk/core';
```

Reply:
Thanks Welly for the feedback. Addressed.